### DISCUSSION TITLE
Image main page

### DISCUSSION THREAD

**Discussion by hopfrogsalerno** [AzuraCast/AzuraCast#4348] [Q&A]
How can I change the default image on the Azuracast home page? Thank you

**Comment by ItsMitchh** [comment]
Custom Branding can be used to change the image.

**Reply by phoenixqss** [reply]
По какому пути я могу найти все изображения для того что бы заменить на свои ? интересует именно путь в докере . Заранее благодарю!

**Reply by ivellios1988** [reply]
@phoenixqss I don't believe @SC2Mitch speaks Russian or whatever Slavic language you write in. If you don't feel comfortable with writing in English on your own then you can always use a translator. For example, this one: https://www.deepl.com/translator

**Comment by ivellios1988** [comment]
Guys you should check out this discussion: https://github.com/AzuraCast/AzuraCast/discussions/4376
